.travis.yml

```diff
@@ -15,6 +15,7 @@ before_script:
     - cmake .. -DCMAKE_BUILD_TYPE=$CONFIGURATION
 
 script:
-    - cmake --build .
-    - ctest -V
-    - qmlscene --quit -I ../qml -P output/ ../examples/TestApp.qml
+    - cmake --build . --config $CONFIGURATION
+    - cmake -P cmake_install.cmake
```

Comment:
I'll do it, but its spammier on the console and less clear for users (i.e. it shows a whole bunch of msbuild noise on windows instead of a neat summary)